ChatRPG/Areas/Identity/RevalidatingIdentityAuthenticationStateProvider.cs

```diff
@@ -28,10 +28,10 @@ public class RevalidatingIdentityAuthenticationStateProvider<TUser>
         AuthenticationState authenticationState, CancellationToken cancellationToken)
     {
         // Get the user manager from a new scope to ensure it fetches fresh data
-        var scope = _scopeFactory.CreateScope();
+        IServiceScope scope = _scopeFactory.CreateScope();
         try
         {
-            var userManager = scope.ServiceProvider.GetRequiredService<UserManager<TUser>>();
+            UserManager<TUser?> userManager = scope.ServiceProvider.GetRequiredService<UserManager<TUser>>();
```

Comment:
Der er to build warnings til linje 34:

`Warning 1: The type 'TUser?' cannot be used as type parameter 'TUser' in the generic type or method 'UserManager<TUser>'. Nullability of type argument 'TUser?' doesn't match 'class' constraint.`

`Warning 2: Nullability of reference types in value of type 'UserManager<TUser>' doesn't match target type 'UserManager<TUser?>'.`